pypowsybl/security.py

```diff
@@ -40,47 +42,18 @@
 class SecurityAnalysisResult:
     def __init__(self, result):
         self.ptr = result
-        results = _pypowsybl.get_security_analysis_result(result)
-        self._post_contingency_results = {}
-        for result in results:
-            if result.contingency_id:
-                self._post_contingency_results[result.contingency_id] = result
-            else:
-                self._pre_contingency_result = result
+        self.results = _create_data_frame_from_series_array(_pypowsybl.get_security_analysis_result(result))
```

Comment:
I think for now we must **also** keep the "old" result, because it carries the information about the validity of the computation for each contingency: if we keep only the violations dataframe we lose it!